app/policies/like_policy.rb

```diff
@@ -0,0 +1,14 @@
+class LikePolicy < ApplicationPolicy
+  attr_reader :user, :like
+
+  def initialize(user, like)
+    @user = user
+    @like = like
+  end
+  def create?
+    true
+  end
+  def show?
+    user == 
```

Comment:
?????

Reply:
totally forgot about this. I was when I try to make the liked photos not showing if the current_user is not following the photo's owner.